src/components/SettingsView.tsx

```diff
@@ -45,50 +45,60 @@ export default function SettingsView () {
       <div className='form' id="themeSelectionForm">
         <label htmlFor="themeSelectionForm">Select Theme:</label>
         <div className='radio'>
-          <input type = "radio"
-            id = "theme"
-            value = "default"
-            checked = {selectedTheme === 'default'}
-            onChange = {handleThemeSelection}
-          />
-          <label>Default (Dark)</label>
+          <label>
```

Comment:
Looks like these were slightly incorrect from an accessibility standpoint before your changes!

So, each of these radio inputs should have their own id (e.g. `id='theme-default'` instead of them all being `id='theme'`).

From there, each `label` should be a sibling element of the `input` (rather than a parent/wrapper), but with a `for` element that refers to the input's id.

e.g. this example should be:
```html
<input type = "radio" value = "default" id = "theme-default" checked = {selectedTheme === 'default'} onChange = {handleThemeSelection} />
<label for='theme-default'>Default (Dark)</label>
```

Reply:
...huh. Okay. Is it fine that it passes the linter now, or is the linter too generous?

Reply:
I think we're fine for now. This feels unidiomatic to me, but after some research it does still produce the correct screen reader behavior.